es-ds-docs/package.json

```diff
@@ -28,7 +28,8 @@
         "nuxt-primevue": "^3.0.0",
         "primevue": "^3.53.0",
         "prismjs": "^1.29.0",
-        "vue": "^3.4.34"
+        "vue": "^3.4.34",
+        "html-truncate": "^1.2.2"
```

Comment:
I think `html-truncate` could be added just to `es-ds-components`?

Reply:
True, removed it from `es-ds-docs`